fxos/js/owners.js

```diff
@@ -104,23 +104,23 @@ var OWNERS = {
   // "Hardware": "?", // Firefox OS
   "Hardware Abstraction Layer (HAL)": "Dave Hylands", // Core
   "ImageLib": "Milan Sreckovic (Jeff Muizelaar, Seth Fowler)",
-  // "Infrastructure": "?",
+  // "Infrastructure": "?", // Firefox OS
   "IPC": "Andrew Overholt (Johnny Stenback)",
   "JavaScript Engine": "Naveed Ihsanullah", // Core
   "JavaScript: GC": "Naveed Ihsanullah", // Core
   "Layout": "Jet Villegas", // Core
-  // "MCTS Waiver Request": "?",
+  // "MCTS Waiver Request": "?", // Firefox OS
+  // "Metrics": "?", // Firefox OS
   "MTP/UMS": "Ben Tian", // Firefox OS
-  "MFBT": "Naveed Ihsanullah",
+  "MTBF": "Naveed Ihsanullah", // Firefox OS
```

Comment:
I don't think there is a `MFBT` comp in any product, but I am not sure of Naveed is responsible of `MTBF`.